app/assets/stylesheets/styles/game.scss

```diff
@@ -76,7 +78,6 @@
   color: #9bb0bb;
 }
 
-// .hover {
-//   -webkit-box-shadow: inset 0px 0px 5px 5px rgba(50, 150, 25, .5);
-//   box-shadow: inset 0px 0px 5px 5px rgba(50, 150, 25, .5);
-// }
+a {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.